src/promptflow/promptflow/entities/__init__.py

```diff
@@ -6,28 +6,9 @@
 
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
 
-from promptflow._sdk.entities._connection import (
-    AzureContentSafetyConnection,
-    AzureOpenAIConnection,
-    CognitiveSearchConnection,
-    CustomConnection,
-    OpenAIConnection,
-    SerpConnection,
-    QdrantConnection,
-    FormRecognizerConnection,
-)
+
 from promptflow._sdk.entities._run import Run
 
 __all__ = [
-    # Connection
-    "AzureContentSafetyConnection",
-    "AzureOpenAIConnection",
-    "OpenAIConnection",
-    "CustomConnection",
-    "CognitiveSearchConnection",
-    "SerpConnection",
-    "QdrantConnection",
-    "FormRecognizerConnection",
-    # Run
     "Run",
```

Comment:
Let's get back the entities, so no breaking to samples

Reply:
Added back